### PR TITLE
SC-15711 add sematext agent release notes page

### PR DESCRIPTION
**Pull request by boratanrikulu**

### CODE REVIEW

**Review by yagogou — APPROVED** [review]

File: docs/agents/sematext-agent/releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+- Swarm housekeeping functionalty.
+
+## Breaking changes
+
+We added StatefulSets and DaemonSets metrics collecting support and this new feature requires some additional cluster roles. If you're using Kubernetes and you installed the agent with `kubectl`, please update the clusterroles before upgrading your agent.
+
+```
+kubectl apply -f https://sematext-installer.s3.amazonaws.com/sematext-clusterroles.yaml
+```
```
[the remark this answers is not in the document]
Comment (boratanrikulu) [reply docs/agents/sematext-agent/releasenotes.md:21]: As I know, no additional steps required for the helm, just regular upgration step. @PabloB94, is it correct?

[the remark this answers is not in the document]
Comment (regiluze) [reply docs/agents/sematext-agent/releasenotes.md:21]: I guess that the idea eventually it's to generate this file automatically, right?

[the remark this answers is not in the document]
Comment (boratanrikulu) [reply docs/agents/sematext-agent/releasenotes.md:21]: Yes, but in agent (sta) repo. This one will be public @regiluze

File: docs/agents/sematext-agent/releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- OpenSearch auto-discovery support. Now you can easily start to collect the metrics of your OpenSearch services. Check [**OpenSearch Monitoring Integration**](https://sematext.com/integrations/opensearch/) for more information.
+- StatefulSets and DaemonSets metrics collecting support.
+
+## Bugfixes
+
+- Swarm housekeeping functionalty.
+
+## Breaking changes
+
+We added StatefulSets and DaemonSets metrics collecting support and this new feature requires some additional cluster roles. If you're using Kubernetes and you installed the agent with `kubectl`, please update the clusterroles before upgrading your agent.
```
Comment (PabloB94) [inline docs/agents/sematext-agent/releasenotes.md:17]: I would add the instruction on how to add the permissions.

```suggestion
We added StatefulSets and DaemonSets metrics collection support. This new feature requires some additional cluster roles. If you're using Kubernetes and you installed the agent with `kubectl`, please update the clusterroles before upgrading your agent by running the following command:

`kubectl apply -f https://sematext-installer.s3.amazonaws.com/sematext-clusterroles.yaml`
```

File: docs/agents/sematext-agent/releasenotes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (PabloB94) [reply docs/agents/sematext-agent/releasenotes.md:21]: The helm chart was updated with the new cluster roles, so a normal helm upgrade should pull those.

File: docs/agents/sematext-agent/releasenotes.md (hunk 2 above)
Comment (PabloB94) [reply docs/agents/sematext-agent/releasenotes.md]: I think I need new glasses 🤦🏽‍♂️

**Pull request merged**